CMakeLists.txt

```diff
@@ -54,6 +54,11 @@ if (WITH_GFLAGS)
     set (HAVE_LIB_GFLAGS 1)
     # When linking unittest, mingw32 cannot find -lgflags without this.
     LINK_DIRECTORIES( ${LINK_DIRECTORIES} ${gflags_INCLUDE_DIR}/../lib)
+    # A deprecated API, but gflags need it.
+    if(WIN32)
+	  set(CMAKE_CXX_STANDARD_LIBRARIES "${CMAKE_CXX_STANDARD_LIBRARIES} -lshlwapi")
+	  set(CMAKE_C_STANDARD_LIBRARIES "${CMAKE_C_STANDARD_LIBRARIES} -lshlwapi")
```

Comment:
Sorry for this error. Only first first pull request is I want, the Github push other two changes to this request automatic. I'm use pull requests in github is unskilled, how can I stop it?